src/articles/dto/article-response.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { Article } from '@prisma/client';
+import { Transform } from 'class-transformer';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { capitalizeTitle } from './articles-response.dto';
+
+export class ArticleResponseDto implements Article {
```

Comment:
I would rename it to `ArticleDetailsResponseDto` to avoid confusing `ArticleResponseDto` and `ArticlesResponseDto`.
We are sending more details here by providing the full `text`.